src/thunderbots/software/ai/navigator/path_planner/straight_line_path_planner.cpp

```diff
@@ -0,0 +1,12 @@
+//
+// Created by jordan on 3/23/19.
+//
```

Comment:
We should switch this to a proper full start-of-file comment

Reply:
there's a start of file comment in the `straight_line_path_planner.h`

Reply:
Technically we're supposed to put it in the `.cpp` and the `.h`. I agree that it's probably reasonable to just start putting it in the `.h` file, but we should at least be consistent for now.

Reply:
added a comment